### PR TITLE
Aura Events

### PR DESCRIPTION
**Pull request by manolama**

### CODE REVIEW

File: events-lucene/src/main/java/net/opentsdb/aura/events/lucene/LuceneQueryBuilder.java [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+    try {
+      customQuery = customParser.parse(query);
+      LOG.info("Original Query = " + customQuery);
+    } catch (ParseException e) {
+      throw new IllegalStateException("Cannot parse the query", e);
+    }
+
+    customQuery = QueryParser.parse(customQuery);
+
+    LOG.info("Yamas Parsed Query = " + customQuery);
```
Comment (anonthing) [inline events-lucene/src/main/java/net/opentsdb/aura/events/lucene/LuceneQueryBuilder.java:100]: remove Yamas

File: events-lucene/src/main/java/net/opentsdb/aura/events/lucene/query/LuceneQueryBuilder.java [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+    try {
+      customQuery = customParser.parse(query);
+      LOG.info("Original Query = " + customQuery);
+    } catch (ParseException e) {
+      throw new IllegalStateException("Cannot parse the query", e);
+    }
+
+    customQuery = QueryParser.parse(customQuery);
+
+    LOG.info("Yamas Parsed Query = " + customQuery);
```
Comment (anonthing) [inline events-lucene/src/main/java/net/opentsdb/aura/events/lucene/query/LuceneQueryBuilder.java:97]: remove Yamas

**Review by anonthing — CHANGES_REQUESTED** [review]

**Review by anonthing — APPROVED** [review]
LGTM! javadoc later.

File: events-common/src/main/java/net/opentsdb/events/EventWriter.java [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+import java.util.concurrent.LinkedBlockingQueue;
+
+public abstract class EventWriter implements Indexer, Runnable {
+
+  private static final Logger LOG = LoggerFactory.getLogger(EventWriter.class);
+  public static DateTimeFormatter df = DateTimeFormatter.ofPattern("yyyy-MM-dd");
+  private static LinkedBlockingQueue<Event> queue = new LinkedBlockingQueue<>(10000);
```
Comment (smrutilal2) [inline events-common/src/main/java/net/opentsdb/events/EventWriter.java:35]: Why is it static?

Comment (smrutilal2) [reply events-common/src/main/java/net/opentsdb/events/EventWriter.java:35]: `10000` should be configurable?

File: events-lucene/build.gradle [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+ */
+
+plugins {
+    id "java-library"
+}
+
+dependencies {
+    api project(':events-common')
+
+    compile 'org.apache.lucene:lucene-core:8.1.1'
```
Comment (smrutilal2) [inline events-lucene/build.gradle:25]: It has mixed the new and old dependencies scopes. Let's use all new and change `compile` to `implementation`

**Comment by smrutilal2** [comment]
Shouldn't this be a different module in itself? Does it have anything to do with Aura?

File: events-common/src/main/java/net/opentsdb/events/EventWriter.java (hunk 3 above)
Comment (anonthing) [reply events-common/src/main/java/net/opentsdb/events/EventWriter.java:35]: I don't think this is used. Same with the `run` in the class.

**Review by smrutilal2 — APPROVED** [review]

**Pull request merged**